Assets/Scripts/Platform Systems/RotatingPlatform.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using DG.Tweening;
+
+public class RotatingPlatform : MonoBehaviour 
+{
+    public GameObject rotatingPlatform;
```

Comment:
Just for fun, can you add a counterClockwise bool, and use that so we can set it to rotate clockwise or counterclockwise? Also instead of MonoBehavior, make it inherit from BasePlatform. 

after that you're good to go!